external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchStream.scala

```diff
@@ -56,7 +56,7 @@ import org.apache.spark.util.UninterruptibleThread
  * and not use wrong broker addresses.
  */
 private[kafka010] class KafkaMicroBatchStream(
-    kafkaOffsetReader: KafkaOffsetReader,
+    private val offsetReader: KafkaOffsetReader,
```

Comment:
- Is `private val` required for `Use CaseInsensitiveMap for KafkaOffsetReader`?
- Is the renaming from `kafkaOffsetReader` to `offsetReader` required for `Use CaseInsensitiveMap for KafkaOffsetReader`?

Reply:
> Is private val required for Use CaseInsensitiveMap for KafkaOffsetReader?

Yeah, please see my previous comment. Same applies here.

> Is the renaming from kafkaOffsetReader to offsetReader required for Use CaseInsensitiveMap for KafkaOffsetReader?
 
In short, no. A little bit detailed if not renamed then the test requires something like this:
```
  private val offsetReaderMicroBatchMethod = PrivateMethod[KafkaOffsetReader]('kafkaOffsetReader)
  private val offsetReaderContinuousMethod = PrivateMethod[KafkaOffsetReader]('offsetReader)
...
    verifyFieldsInMicroBatchStream(KafkaSourceProvider.FETCH_OFFSET_NUM_RETRY, expected, stream => {
      val kafkaOffsetReader = getField(stream, offsetReaderMicroBatchMethod)
      assert(expected.toInt === getField(kafkaOffsetReader, fetchOffsetNumRetriesMethod))
    })
...
```
General considerations why I've renamed couple of variables:
* Exact same parameters are named sometimes 2-4 different ways which is hard to track
* Not renaming them would cause similar situation in the test code
* Sometimes the config name and it's internal variable has different name
* Renamed only Spark internal variables which shouldn't matter for users

We can roll back renamings, the code will work properly but I think these changes increase readability.

Reply:
The rename rolled back.
`private val` I think is essential.